include/flatbuffers/flatbuffers.h

```diff
@@ -283,6 +283,11 @@ template<typename T> class Vector {
     return reinterpret_cast<const void *>(Data() + o);
   }
 
+  template<typename S>
+  const void *GetAsStruct(const S &sd, uoffset_t i) const {
+    return reinterpret_cast<const void *>(Data() + sd.bytesize * i);
```

Comment:
This relies on `bytesize`, which is only available to schema objects, so a bit weird for the main `flatbuffers.h` to depend on it? Since its only used in `idl_gen_text.cpp`, maybe move it there?

Reply:
Yes, this method is not necessary, I will remove it.
Initially, I thought about using a method `Vector::GetStructFromOffset`, but added `GetAsStruct` with indexed access as a little bit more safe method.